src/app/index.js

```diff
@@ -294,6 +303,12 @@ export default yeoman.generators.Base.extend({
 
     this._currentPkgJSON = this.fs.readJSON(this.destinationPath('package.json'), null);
 
+    this._ghooksOptions = {
+      lint: 'Check code quality using videojs-standard',
+      test: 'Run unit tests using Karma',
```

Comment:
We may want specify that test will also lint due to the `pretest` command in `package.json`